File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/UploadOfflineSessionsResponse.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

public class UploadOfflineSessionsResponse extends BaseResponseContent {
    @SerializedName("content")
    public OfflineSessionsBaseResponseContent content;

    @SerializedName("error")
    public ResponseError error;
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/UploadOfflineSessionsResponse.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class UploadOfflineSessionsResponse : BaseResponseContent() {

    @SerializedName("content")
    var content: OfflineSessionsBaseResponseContent? = null

    @SerializedName("error")
    var error: ResponseError? = null

}